src/Microsoft.ML.Ensemble/Trainer/EnsembleTrainerBase.cs

```diff
@@ -54,27 +54,22 @@ public abstract class ArgumentsBase : LearnerInputBaseWithLabel
             public bool ShowMetrics;
 
             [Argument(ArgumentType.Multiple, HelpText = "Base predictor type", ShortName = "bp,basePredictorTypes", SortOrder = 1, Visibility = ArgumentAttribute.VisibilityType.CmdLineOnly)]
-            public SubComponent<ITrainer<RoleMappedData, IPredictorProducing<TOutput>>, TSig>[] BasePredictors;
+            public SubComponent<ITrainer<IPredictorProducing<TOutput>>, TSig>[] BasePredictors;
         }
 
         private const int DefaultNumModels = 50;
         /// <summary> Command-line arguments </summary>
-        protected readonly ArgumentsBase Args;
-        protected readonly int NumModels;
+        protected internal readonly ArgumentsBase Args;
```

Comment:
Why are these fields all marked as `protected internal` now?  Looking through the code, I don't see anyone outside of the class actually using them, and I would assume they would be `private`.

If we want to expose them outside of the class, should they be properties instead?

Reply:
Sorry @eerhardt ... blunder on my part, I got confused between that and `private protected`, everywhere I introduced `protected internal` I really meant for it to be `private protected`. ☹️